docs/UserGuide.md

```diff
@@ -202,537 +237,669 @@ This subsection serves to list out the parameters used in this document. To achi
 * Existing values will be updated to the input values.
+
+<div markdown="block" class="alert alert-info">:information_source: **Info:**
+
+* At least one of the optional fields must be provided.
 * When updating roles, the existing roles of the groupmate will be removed i.e. adding of roles is not cumulative.
-* You can remove all the roles of the groupmate by typing `r/` without
-  specifying any roles after it.
+* You can remove all the roles of the groupmate by typing `r/` without specifying any roles after it.
+
+</div>
+
+Parameters:
+
+* [`PROJECT_INDEX`](#index) The index of the project.
+* [`GROUPMATE_INDEX`](#index) The index of the groupmate.
+* [`NAME`](#name) The name of the groupmate.
+* [`ROLE`](#role) The role of the groupmate.
 
 Examples:
-*  `updateG 1 i/1 p/91234567 n/Leslie Knope` Updates the name of the 1st groupmate of the 1st project to be `Leslie Knope`.
-*  `updateG 2 i/2 n/Betsy Crower r/` Updates the name of the 2nd groupmate of the 2nd project to be `Betsy Crower` and clears all existing roles.
-*  `updateG 2 i/3 n/Betsy Crower r/group-leader r/tester` Updates the name of the 3rd groupmate of the 1st project to be `Betsy Crower` and changes her roles to `group-leader` and `tester`.
 
-#### Deleting a groupmate from a project: `deleteG`
+* `updateG 1 i/1 n/Leslie Knope` Updates the name of the first groupmate of the first project with the new name `Leslie Knope`.
+* `updateG 2 i/2 n/Betsy Crower r/` Updates the name of the second groupmate of the second project with the new name `Betsy Crower` and clears all existing roles.
+* `updateG 2 i/3 n/Betsy Crower r/group-leader r/tester` Updates the name of the third groupmate of the first project to the new name `Betsy Crower` and changes her roles to `group-leader` and `tester`.
+
+##### Deleting a groupmate from a project: `deleteG`
 
-Deletes a specified groupmate from a specified project.
+Deletes a groupmate from a project in CoLAB.
 
 Format: `deleteG PROJECT_INDEX i/GROUPMATE_INDEX`
 
 * Deletes the groupmate at the specified `GROUPMATE_INDEX` from the project at the specified `PROJECT_INDEX`.
-* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
-* The `GROUPMATE_INDEX` refers to the number shown beside the groupmate when viewing the project.
-* Both indexes **must be positive integers** 1, 2, 3, …​
 
-Examples:
-* `deleteG 1 i/1`
-* `deleteG 2 i/2`
+Parameters:
+
+* [`PROJECT_INDEX`](#index) The index of the project.
+* [`GROUPMATE_INDEX`](#index) The index of the groupmate.
+
+Example:
+
+* `deleteG 1 i/1` deletes the first groupmate of the first project.
 
 ### **4.2 Contacts**
 
-Contacts are kept separate from Groupmates. Their information is not kept in sync. 
+Contact are used in CoLAB to store contact information about a person, to help you keep track of people you contact frequently.
```

Comment:
Here too. "Maybe this portion can be clearer on how Contacts and Groupmates are still separate entities."